cloudbuild.yaml

```diff
@@ -291,27 +291,32 @@ steps:
       set -o pipefail
       pids=()
       cloudProducts=("generic" "gke-autopilot")
+      k8sVersions=("1.23" "1.24" "1.25")
+      clusterRegions=("us-east1" "us-west1" "us-central1")
```

Comment:
When we add support for 1.26 and drop 1.23, if we don't shift this array to keep it aligned with the k8sVersions array then we are going to shift the region in which we are looking for the test cluster location. If these are going to be tightly coupled, then I wonder if we can stick them into the same data structure as pairs (e.g. `({version1, region1}, {version2, region2}, ...)`). WDYT?

Reply:
Totally makes sense. I'm making them pairs.